csharp/test/Ice/ami/TestI.cs

```diff
@@ -127,17 +127,13 @@ public override bool
                 return true;
             }
 
-            public override async Task
-            opAsyncDispatchAsync(Ice.Current current)
-            {
-                await Task.Delay(10);
-            }
+            public override Task opAsyncDispatchAsync(Ice.Current current) => Task.CompletedTask;
```

Comment:
Shouldn't we use Task.Yield here too, to test asynchronous completion?

Reply:
As far as I can tell, that's not what this test is testing.